docs/DeveloperGuide.adoc

```diff
@@ -358,14 +358,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *`   |Forgetful student | Set reminders for upcoming events | Will remember to attend them
 
+|`* * *` |Student | Add tasks to study plan | Can add study tasks to my study plan
+
+|`* * *` |Student | Delete tasks on study plan | Can delete study tasks if I don't need it anymore
+
+|`* * *` |Student | Auto-generate event from a task | If I feel like I want to do a study task today, I can generate an event on today's schedule, or any day.
```

Comment:
A student cannot auto-generate event because it is the app's function. Maybe can remove the auto?
The last column doesn't match the header ("So that I...). Maybe "Can set a time to complete the study task." would be better?